cpp/ql/src/semmle/code/cpp/security/Security.qll

```diff
@@ -70,7 +70,7 @@ class SecurityOptions extends string {
    */
   predicate userInputArgument(FunctionCall functionCall, int arg) {
     exists(string fname |
-      functionCall.getTarget().hasGlobalName(fname) and
+      functionCall.getTarget().hasGlobalOrStdName(fname) and
       exists(functionCall.getArgument(arg)) and
       (
         fname = "read" and arg = 1
```

Comment:
`read` is a POSIX function and belongs with the other non-`std` functions below.